razor/passes.py

```diff
@@ -55,7 +55,7 @@ def interface(input_file, output_file, wrt, use_seadsa):
     args = ['-Pinterface']
     if use_seadsa:
         args += ['-Pinterface-with-seadsa',
-                 # improve precision of sea-dsa by considering types            
+                 # improve precision of sea-dsa by considering types
```

Comment:
Similar to Rafae's PR. Your editor is adding extra white spaces that make hard to review the PR